tools/protovalidate-migrate/internal/migrator/field.go

```diff
@@ -136,7 +139,8 @@ func (v *FieldVisitor) printComma(printComments bool, commas ...ast.Node) error
 	return err
 }
 
-func (v *FieldVisitor) buildNameParts(node *ast.OptionNode) (nameParts []ast.Node, done bool) {
+func (v *FieldVisitor) buildNameParts(node *ast.OptionNode) (nameParts []ast.Node, value ast.Node, done bool) {
```

Comment:
Any reason this needs to use named return values? I think you could do this to avoid them:
```go
value := ast.Node(node.Val)
nameParts := make([]ast.Node, 1, len(node.Name.Parts)*2+1)
```